vllm/v1/worker/gpu_model_runner.py

```diff
@@ -681,7 +685,7 @@ def _prepare_inputs(
         scheduler_output: "SchedulerOutput",
     ) -> tuple[dict[str,
                     Any], bool, torch.Tensor, Optional[SpecDecodeMetadata],
-               np.ndarray, Optional[CommonAttentionMetadata]]:
+               np.ndarray, Optional[CommonAttentionMetadata], int]:
```

Comment:
Could we add the int to the docstring below? I assume this is the `max_query_len`?

Reply:
Nvm I see it's `max_num_scheduled_tokens`